packages/data-context/src/data/ProjectConfigIpc.ts

```diff
@@ -157,7 +157,13 @@ export class ProjectConfigIpc extends EventEmitter {
 
       let resolved = false
 
-      this._childProcess.on('error', (err) => {
+      this._childProcess.on('error', (err: NodeJS.ErrnoException) => {
+        if (err.code === 'EPIPE') {
+          debug('EPIPE error in loadConfig() of child process %s', err)
+
```

Comment:
@alexsch01 I wonder if we just want to `resolve()` this before returning just to prevent a possible memory leak (even though the process _should_ be killed)
```suggestion
resolve()
```